Backpack/Icon/Classes/BPKIcon.m

```diff
@@ -125,6 +123,19 @@ + (UIImage *)iconNamed:(NSString *)name color:(UIColor *)color size:(BPKIconSize
     return icon;
 }
 
++(NSString *)codepointForIcon:(NSString *)name size:(BPKIconSize)size {
+    NSString *lookupKey = name;
+    if(size == BPKIconSizeSmall) {
+        lookupKey = [NSString stringWithFormat:@"%@%@", name, @"-sm"];
+    }
+
+    if (self.iconMapping) {
+        NSAssert([self.iconMapping objectForKey:lookupKey], @"Unknown icon `%@` does not correspond to a known icon. Please check that the icon exists at the size you are trying to use https://backpack.github.io/components/icon?platform=design", lookupKey);
```

Comment:
This might result in confusing errors because the `-sm` suffix is an implementation detail that the caller might not be aware of. It might be better to change the message to metnion the basename and the size requested